games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,104 @@
+SUMMARY="Free 3D go-kart racing game"
+DESCRIPTION="Karts. Nitro. Action! SuperTuxKart is a 3D open-source arcade racer \
```

Comment:
Wrap after 80th character.

Reply:
Still not resolved, check https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description